core/templates/dev/head/components/summary_tile/ExplorationSummaryTileDirective.js

```diff
@@ -43,7 +43,9 @@ oppia.directive('explorationSummaryTile', [
         // version of the summary tile is displayed. This attribute is optional:
         // if it is not specified, it is treated as 0, which means that the
         // desktop version of the summary tile is always displayed.
-        mobileCutoffPx: '@mobileCutoffPx'
+        mobileCutoffPx: '@mobileCutoffPx',
+        isPlaylistMode: '&playlistMode',
```

Comment:
Ditto here and next line re naming of attrs.

Reply:
Done! And I had changed this and pushed it. Don't know why that happened.